The `nixpkgs-review` is resulting in an error (but `nix build .#meteor` works):

```
$ git worktree add /home/manu/.cache/nixpkgs-review/rev-8af3fc4c7970ed68828b88d548bcd0db17a4f7ce/nixpkgs 263d8f7fcef914ff43c6e8445451f18798188ed0
Preparing worktree (detached HEAD 263d8f7fcef)
Updating files: 100% (34710/34710), done.
HEAD is now at 263d8f7fcef Merge pull request #228675 from fabaff/bashlex-bump
$ nix-env --option system x86_64-linux -f /home/manu/.cache/nixpkgs-review/rev-8af3fc4c7970ed68828b88d548bcd0db17a4f7ce/nixpkgs -qaP --xml --out-path --show-trace --no-allow-import-from-derivation
$ git merge --no-commit --no-ff 8af3fc4c7970ed68828b88d548bcd0db17a4f7ce
Automatic merge went well; stopped before committing as requested
$ nix-env --option system x86_64-linux -f /home/manu/.cache/nixpkgs-review/rev-8af3fc4c7970ed68828b88d548bcd0db17a4f7ce/nixpkgs -qaP --xml --out-path --show-trace --no-allow-import-from-derivation --meta
error:
       … while evaluating call site

       at /home/manu/.cache/nixpkgs-review/rev-8af3fc4c7970ed68828b88d548bcd0db17a4f7ce/nixpkgs/pkgs/top-level/all-packages.nix:25295:12:

        25294|
        25295|   meteor = callPackage ../servers/meteor { };
             |            ^
        25296|

       … while calling 'callPackageWith'

       at /home/manu/.cache/nixpkgs-review/rev-8af3fc4c7970ed68828b88d548bcd0db17a4f7ce/nixpkgs/lib/customisation.nix:128:35:

          127|   */
          128|   callPackageWith = autoArgs: fn: args:
             |                                   ^
          129|     let

       error: evaluation aborted with the following error message: 'Function called without required argument "lzma" at /home/manu/.cache/nixpkgs-review/rev-8af3fc4c7970ed68828b88d548bcd0db17a4f7ce/nixpkgs/pkgs/servers/meteor/default.nix:1, did you mean "lima", "czmq" or "emma"?'
$ git worktree prune
Traceback (most recent call last):
  File "/nix/store/7cmyqf2hzimhbavnp7glp5jig18inndl-nixpkgs-review-2.8.0/bin/.nixpkgs-review-wrapped", line 9, in <module>
    sys.exit(main())
  File "/nix/store/7cmyqf2hzimhbavnp7glp5jig18inndl-nixpkgs-review-2.8.0/lib/python3.10/site-packages/nixpkgs_review/__init__.py", line 10, in main
    cli.main(command, args)
  File "/nix/store/7cmyqf2hzimhbavnp7glp5jig18inndl-nixpkgs-review-2.8.0/lib/python3.10/site-packages/nixpkgs_review/cli/__init__.py", line 301, in main
    return cast(str, args.func(args))
  File "/nix/store/7cmyqf2hzimhbavnp7glp5jig18inndl-nixpkgs-review-2.8.0/lib/python3.10/site-packages/nixpkgs_review/cli/rev.py", line 12, in rev_command
    return review_local_revision(f"rev-{commit}", args, commit)
  File "/nix/store/7cmyqf2hzimhbavnp7glp5jig18inndl-nixpkgs-review-2.8.0/lib/python3.10/site-packages/nixpkgs_review/review.py", line 497, in review_local_revision
    review.review_commit(builddir.path, args.branch, commit, staged)
  File "/nix/store/7cmyqf2hzimhbavnp7glp5jig18inndl-nixpkgs-review-2.8.0/lib/python3.10/site-packages/nixpkgs_review/review.py", line 263, in review_commit
    self.start_review(self.build_commit(branch_rev, reviewed_commit, staged), path)
  File "/nix/store/7cmyqf2hzimhbavnp7glp5jig18inndl-nixpkgs-review-2.8.0/lib/python3.10/site-packages/nixpkgs_review/review.py", line 156, in build_commit
    merged_packages = list_packages(
  File "/nix/store/7cmyqf2hzimhbavnp7glp5jig18inndl-nixpkgs-review-2.8.0/lib/python3.10/site-packages/nixpkgs_review/review.py", line 335, in list_packages
    subprocess.run(cmd, stdout=tmp, check=True)
  File "/nix/store/iw1vmh509hcbby8dbpsaanbri4zsq7dj-python3-3.10.10/lib/python3.10/subprocess.py", line 526, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['nix-env', '--option', 'system', 'x86_64-linux', '-f', '/home/manu/.cache/nixpkgs-review/rev-8af3fc4c7970ed68828b88d548bcd0db17a4f7ce/nixpkgs', '-qaP', '--xml', '--out-path', '--show-trace', '--no-allow-import-from-derivation', '--meta']' returned non-zero exit status 1.
```

Not sure if this is actually a problem with my changes or something else :confused:

--- a/pkgs/servers/meteor/default.nix
+++ b/pkgs/servers/meteor/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, lib, fetchurl, zlib, patchelf, runtimeShell }:
+{ stdenv, lib, fetchurl, zlib, curl, xz, patchelf, runtimeShell }:
 
 let
   version = "2.7.3";
@@ -66,7 +66,7 @@ stdenv.mkDerivation {
     popd
     substituteInPlace $out/tools/cli/main.js \
       --replace "@INTERPRETER@" "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-      --replace "@RPATH@" "${lib.makeLibraryPath [ stdenv.cc.cc zlib ]}" \
+      --replace "@RPATH@" "${lib.makeLibraryPath [ stdenv.cc.cc zlib curl xz ]}" \
       --replace "@PATCHELF@" "${patchelf}/bin/patchelf"
 
     # Patch node.
@@ -79,7 +79,7 @@ stdenv.mkDerivation {
     for p in $out/dev_bundle/mongodb/bin/mongo{,d}; do
       patchelf \
         --set-interpreter $(cat $NIX_CC/nix-support/dynamic-linker) \
-        --set-rpath "$(patchelf --print-rpath $p):${lib.makeLibraryPath [ stdenv.cc.cc zlib ]}" \
+        --set-rpath "$(patchelf --print-rpath $p):${lib.makeLibraryPath [ stdenv.cc.cc zlib curl xz ]}" \
         $p
     done
 

